content/applications/finance/fiscal_localizations/colombia/configuration.rst

```diff
@@ -0,0 +1,132 @@
+=============
+Configuration
+=============
+
+.. _fiscal_localizations/colombia/installation:
+
+Modules installation
+====================
+
+:ref:`Install <general/install>` the following modules to get all the features of the Colombian
+localization:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 25 25 50
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Colombia - Accounting`
+     - `l10n_co`
+     - Default :ref:`fiscal localization package <fiscal_localizations/packages>`
+   * - :guilabel:`Colombia E-invoice Integration`
+     - `l10n_co_edi`
+     - Carvajal e-invoicing integration
+
+.. _fiscal_localizations/colombia/carvajal:
+
+Configure credentials for Carvajal web service
+==============================================
+
+Once the modules are installed, the user credentials need to be configured in order to connect with
+Carvajal Web Service. First, navigate to :menuselection:`Accounting --> Configuration --> Settings`
+and look for the :guilabel:`Colombian Electronic Invoice` section. Then, fill in the required
+configuration information provided by Carvajal.
+
+.. image:: configuration/carvajal-credential-config.png
+   :align: center
+   :alt: Configure credentials for Carvajal web service in Odoo.
+
+Check the :guilabel:`Test mode` checkbox to connect with the Carvajal testing environment. This
+allows users to test the complete workflow and integration with the :abbr:`CEN (Centro Electrónico
+de Negocios)` Financiero portal, which is accessible here:
+
+`CTS (Carvajal T&S) <https://cenflab.cen.biz/site/>`_.
+
+`CSC (Carvajal Servicios de Comunicación) <https://web-stage.facturacarvajal.com/>`_.
+
+:abbr:`CSC (Carvajal Servicios de Comunicación)` is the default for new databases.
+
+Once Odoo and Carvajal are fully configured and ready for production, the testing environment can
+be disabled by unchecking the :guilabel:`Test mode` checkbox.
+
+.. _fiscal_localizations/colombia/configure_report:
+
+Configure report data
+=====================
+
+Report data can be defined for the fiscal section and the bank information in the PDF as part of
+the configurable information that is sent in the XML.
```

Comment:
double check 100th char breaks throughout all documents this is second one I caught in quick succession.
won't list any more from here.
```suggestion
Report data can be defined for the fiscal section and the bank information in the PDF as part of the
configurable information that is sent in the XML.
```